app/views/cart/_cart.html.erb

```diff
@@ -0,0 +1,28 @@
+<% @carts.each do |(proId, qty)| %>
```

Comment:
Render partial mà sao còn for each kiểu này e?

Reply:
ko càn for nhé